base/server/src/main/java/org/dogtagpki/server/cli/SubsystemIdGeneratorUpdateCLI.java

```diff
@@ -51,16 +48,22 @@ public SubsystemIdGeneratorUpdateCLI(CLI parent) {
     @Override
     public void createOptions() {
         options.addOption("t", "type", true, "Generator type to update.");
+        options.addOption("r", "range", true, "Name of the ranges entry in DS.");
     }
 
     @Override
     public void execute(CommandLine cmd) throws Exception {
+        String newRangesName = "ranges_v2";
```

Comment:
I think ideally the default value should be set depending on the generator type, for example:
```
if (generator == LEGACY_2) {
    newRangesName = "ranges_v2";
}
```
But since we only use this for SSNv2 it doesn't really matter.